<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_loadingLastGame">
       <property name="text">
        <string>加载上一局游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_settingBackgroupPicture">
       <property name="text">
        <string>设置背景图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_voiceEffect">
       <property name="text">
        <string>音效</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_setting">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_help">
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>33</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>中国象棋</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
